source/components/SimplePage/index.js

```diff
@@ -1,34 +1,14 @@
 import React from 'react'
-import { connect } from 'react-redux'
+
 import Main from '../../layouts/Main'
 import styles from './styles.css'
 
-export default connect(
-  ({ pages }) => ({ pages })
-)(({ pages, params }) => {
-  const page = pages[params.splat || '/'] || {}
-  const { content } = page
-
-  if (!content) {
-    return (
-      <Main title="404">
-        <section className={ styles.SimplePage }>
-          <div>Whoops!</div>
-        </section>
-      </Main>
-    )
-  }
-
-  const { title, body } = content
-
-  return (
-    <Main title={ title }>
-      <section className={ styles.SimplePage }>
-        <div dangerouslySetInnerHTML={{
-          __html: body
-        }} />
-      </section>
-    </Main>
-  )
-})
-
+export default ({ title, body }) => (
+  <Main title={ title }>
+    <section className={ styles.SimpleContent }>
+      <div dangerouslySetInnerHTML={{
```

Comment:
This is markdown content unfortunately. We _could_ find something which transforms markdown > react...